static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -122,7 +122,9 @@ export default function FeedbackItem({feedbackItem, eventData, tags}: Props) {
 
         <Section icon={<IconLink size="xs" />} title={t('Url')}>
           <ErrorBoundary mini>
-            <TextCopyInput size="sm">{url?.value ?? t('URL not found')}</TextCopyInput>
+            <TextCopyInput size="sm">
+              {eventData?.tags ? (url ? url.value : t('URL not found')) : ''}
```

Comment:
Are we using `eventData.tags` as loading state? Is there something more generic for loading state? Otherwise, an intermediate var would be helpful here for future readers since it looks like a potential bug.

Reply:
we can, but it'll  make the whole right side load a lot slower since we'll have to wait for all of the event data to load before we display anything